projects/plugins/jetpack/modules/google-fonts/current/load-google-fonts.php

```diff
@@ -18,6 +18,22 @@
  * @return object[] The collection data of the Google Fonts.
  */
 function jetpack_get_google_fonts_data() {
+	/**
+	 * Filters the Google Fonts data before the default retrieval process.
+	 *
+	 * This filter allows short-circuiting the default Google Fonts data retrieval process.
+	 * Returning a non-null value from this filter will bypass the default retrieval
+	 * and return the filtered value instead.
+	 *
+	 * @since TBD
```

Comment:
```suggestion
	 * @since $$next-version$$
```

The `$$next-version$$` tag is automatically replaced by the correct version number on release:
https://github.com/Automattic/jetpack/blob/d3f58235992e84fcc8a4b1bc191c56ba6edf213a/projects/packages/README.md#L116-L123